frontend/src/components/Header.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+  import moment from 'moment';
+  import daysState from '../state/days';
+  import columnStore from '../state/columns';
+
+  function getWeekday(dateAsString) {
+    return moment(dateAsString).format('ddd');
+  }
+
+  function getMonthDay(dateAsString) {
+    return moment(dateAsString).format('MM-DD');
+  }
+
+  $: columnDays = $daysState.length > 0 ? $daysState.slice(0, $columnStore) : [];
+</script>
+
+<div class="container">
+  <div>
+    <p>Room</p>
+  </div>
+
+  {#each columnDays as day, index}
+    <div >
+      <p>{getMonthDay(day)}</p>
+      <p>{getWeekday(day)}</p>
+    </div>
+  {/each}
+</div>
+
+<style>
+	.container {
+		background-color: white;
+		border-bottom: var(--table-view-border);
+    display: grid;
```

Comment:
currently tabs and spaces are mixed, in my opinion only one of both should be used, I'm fine with either!

Reply:
Adjusted to match the rest of the files